test/parallel/test-tls-timeout-server-2.js

```diff
@@ -1,32 +1,32 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
 if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
-var tls = require('tls');
+const tls = require('tls');
 
-var fs = require('fs');
+const fs = require('fs');
 
-var options = {
+const options = {
   key: fs.readFileSync(common.fixturesDir + '/keys/agent1-key.pem'),
   cert: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem')
 };
 
-var server = tls.createServer(options, function(cleartext) {
-  var s = cleartext.setTimeout(50, function() {
+const server = tls.createServer(options, common.mustCall(function(cleartext) {
+  const s = cleartext.setTimeout(50, common.mustCall(function() {
```

Comment:
In that case, would you prefer we add an assertion inside as well? It seems to me that if this test exits prior to cleanup, that indicates that something is wrong.